packages/database/src/lib/queries.ts

```diff
@@ -949,219 +809,70 @@ export const getNodesInRelations = async ({
-  pagination = { limit: 100, offset: 0 },
-}: GetConceptsParams): Promise<PConceptFull[]> => {
-  // Convert new API to old API parameters
-  const oldParams: Parameters<typeof getConceptsLegacy>[0] = {
-    supabase,
-    spaceId,
-    nodeAuthor: author,
-    conceptFields: fields.concepts,
-    contentFields: fields.content,
-    documentFields: fields.documents,
-    relationFields: fields.relations,
-    relationSubNodesFields: fields.relationNodes,
-    limit: pagination.limit,
-    offset: pagination.offset,
-  };
-
-  // Map scope to old parameters
-  switch (scope.type) {
-    case "all":
-      oldParams.schemaLocalIds = [];
-      oldParams.fetchNodes = null;
-      break;
-    case "nodes":
-      oldParams.schemaLocalIds = scope.nodeTypes || [];
-      oldParams.fetchNodes = true;
-      break;
-    case "relations":
-      oldParams.schemaLocalIds = [];
-      oldParams.fetchNodes = false;
-      break;
-    case "schemas":
-      oldParams.schemaLocalIds = NODE_SCHEMAS;
-      oldParams.fetchNodes = null;
-      break;
-    case "specific":
-      oldParams.baseNodeLocalIds = scope.nodeIds || [];
-      oldParams.schemaLocalIds = [];
-      oldParams.fetchNodes = null;
-      break;
-  }
 
-  // Map relation filters to old parameters
-  if (relations) {
-    if (relations.ofType) {
-      oldParams.inRelsOfTypeLocal = relations.ofType;
-    }
-    if (relations.toNodeTypes) {
-      oldParams.inRelsToNodesOfTypeLocal = relations.toNodeTypes;
-    }
-    if (relations.authoredBy) {
-      oldParams.inRelsToNodesOfAuthor = relations.authoredBy;
-    }
-    if (relations.containingNode) {
-      oldParams.inRelsToNodeLocalIds = [relations.containingNode];
-    }
-    if (relations.containingNodes) {
-      oldParams.inRelsToNodeLocalIds = relations.containingNodes;
-    }
-  }
-
-  return getConceptsLegacy(oldParams);
-};
+// instrumentation for benchmarking
+export const LAST_QUERY_DATA = { duration: 0 };
 
-/**
- * Main function for querying concepts with the new grouped parameter API.
- * Provides better developer experience with logical parameter grouping and comprehensive IntelliSense.
- *
- * @param params - Query parameters with grouped structure
- * @returns Promise resolving to array of concept objects with full metadata
- *
- * @example
- * ```typescript
- * // Query nodes of specific types with relation filters
- * const results = await getConcepts({
- *   supabase,
- *   spaceId: 123,
- *   scope: { type: "nodes", nodeTypes: ["page", "note"] },
- *   relations: { ofType: ["cites", "references"] },
- *   author: "user123",
- *   pagination: { limit: 50, offset: 0 }
- * });
- *
- * // Query relations containing specific nodes
- * const relations = await getConcepts({
- *   supabase,
- *   spaceId: 123,
- *   scope: { type: "relations" },
- *   relations: { containingNodes: ["node-123", "node-456"] }
- * });
- * ```
- */
+// Main entry point to query Concepts and related data:
```

Comment:
Any particular reason why the JSDoc and `@example` was removed for `getConcepts`?

Reply:
No, that was an inadvertent loss in the rewrite, apologies. Reinstated and expanded.